dts/dts-tests.robot

```diff
@@ -0,0 +1,220 @@
+*** Settings ***
+Library             Collections
+Library             OperatingSystem
+Library             Process
+Library             String
+Library             Telnet    timeout=20 seconds    connection_timeout=120 seconds
+Library             SSHLibrary    timeout=90 seconds
+Library             RequestsLibrary
+Resource            ../keywords.robot
+Resource            ../keys.robot
+Resource            ../rtectrl-rest-api/rtectrl.robot
+Resource            ../sonoff-rest-api/sonoff-api.robot
+Resource            ../variables.robot
+
+Suite Setup         Run Keywords
+...                     Prepare Test Suite    AND
+...                     Skip If    not ${DTS_SUPPORT}
+Suite Teardown      Run Keyword
+...                     Log Out And Close Connection
+
+
+*** Variables ***
+${DTS_CHECKPOINT}=      Enter an option:
+
+
+*** Test Cases ***
+PC Engines DPP initial deployment (legacy -> UEFI) - no credentials
+    [Documentation]    Verify DPP (coreboot + UEFI) initial deployment logic on
+    ...    PC Engines. We start from legacy firmware and insert no DPP keys. No
+    ...    firmware shall be offered, but we expect link to shop.
+
+    Power On And Enter DTS Shell
+
+    Execute Command In Terminal
+    ...    export BOARD_VENDOR="PC Engines" SYSTEM_MODEL="APU2" BOARD_MODEL="APU2" SYSTEM_VENDOR="PC Engines"
```

Comment:
I am not sure how it works. Ok, you will set globally these variables before executing DTS scripts, but the variables are being overwritten in `dts-environment.sh`, which is being sourced in every script.

Reply:
I see, that in `dts-environment` some variables assignment checks if the variables already has a value:

```bash
BIOS_VENDOR="${BIOS_VENDOR:-$(dmidecode -s bios-vendor)}"
BIOS_VERSION="${BIOS_VERSION:-$(dmidecode -s bios-version)}"
``` 

So, this question is closed.